src/tesar/__main__.py

```diff
@@ -1,25 +1,5 @@
 #!/usr/bin/env python3
 
-import contextlib
-import sys
+from . import main
```

Comment:
Unfortunately not. The thing is that when the package is executed, the `__name__` variable in the `__init__.py` is not `__main__` as the file that's being "executed" is the `__main__.py`. I've just tried it.

Sidenote for that: Even if it worked, it would feel to me more like a non-transparent side-effect having the main function started from different file that's being executed (during the import).

I understand what you probably want to achieve here (having the smallest or no `__main__.py`) but I believe that the `__main__.py` will be no longer needed to be maintained after (and if) this PR is merged and the only thing that would need to be changed is the import statement should the main function be moved elsewhere.